tests/testthat/test-migrate.R

```diff
@@ -1,7 +1,9 @@
 context("migrations")
 
-
 test_that("0.3.2 -> 0.3.3", {
+  oo <- options(orderly.nowarnings = TRUE)
```

Comment:
Why are suppressing warnings in these tests?

Reply:
The old style database configuration that is used there (they are reference examples so need to retain them) trigger warnings, but that's expected here